packages/libs/sdk/README.md

```diff
@@ -0,0 +1,154 @@
+# QuickNode SDK
+
+An SDK from [QuickNode](https://www.quicknode.com/) making it easy for developers to interact with the blockchain.
+
+Currently supports getting started with [Icy Tools GraphQL API](https://developers.icy.tools/) in a blink!
+
+[![npm](https://img.shields.io/npm/dm/@quicknode/sdk)](https://www.npmjs.com/package/@quicknode/sdk)
+[![npm](https://img.shields.io/npm/v/@quicknode/sdk?color=g)](https://www.npmjs.com/package/@quicknode/sdk)
+![Maintenance](https://img.shields.io/maintenance/yes/2022?color=g)
+[![License](https://img.shields.io/npm/l/@quicknode/sdk?color=g)](https://github.com/quiknode-labs/qn-oss/blob/main/LICENSE.txt)
+[![GitHub issues](https://img.shields.io/github/issues-raw/quiknode-labs/qn-oss?color=g)](https://github.com/quiknode-labs/qn-oss/issues)
+[![Discord](https://img.shields.io/discord/880505845090250794?color=g)](https://discord.gg/DkdgEqE)
+
+## Quick Start
+
+```ts
+import { QuickNodeSDK } from '@quicknode/sdk';
+
+const client = new QuickNodeSDK({ gqlApiKey: 'my-api-key' });
+
+client.nft
```

Comment:
What's the expected behavior when not sending the defaultchain?

Reply:
Good call out, I added some information to the docs, lmk if it makes sense